sqlglot/tokens.py

```diff
@@ -341,8 +341,10 @@ class TokenType(AutoName):
     SOME = auto()
     SORT_BY = auto()
     START_WITH = auto()
+    STARTS_WITH = auto()
     STRUCT = auto()
     TABLE_SAMPLE = auto()
+    TERSE = auto()
```

Comment:
Get rid of these two tokens, I don't think we need them. Use `_match_text_seq` instead in the parser.

Reply:
I've removed `TERSE` but I'm not sure how to remove the `STARTS WITH` token ... if i do the `STARTS` ends up as `VAR` and is consumed as the database name, 

eg:
```
show terse schemas in database db1 starts with 'a' limit 10 from 'b'` I get:
```

without a `STARTS WITH` token is parsed as:
```
[
    <Token token_type: TokenType.SHOW, text: show, line: 1, col: 4, start: 0, end: 3, comments: []>, 
    <Token token_type: TokenType.VAR, text: terse, line: 1, col: 10, start: 5, end: 9, comments: []>,
    <Token token_type: TokenType.VAR, text: schemas, line: 1, col: 18, start: 11, end: 17, comments: []>,
    <Token token_type: TokenType.IN, text: in, line: 1, col: 21, start: 19, end: 20, comments: []>,
    <Token token_type: TokenType.DATABASE, text: database, line: 1, col: 30, start: 22, end: 29, comments: []>,
    <Token token_type: TokenType.VAR, text: db1, line: 1, col: 34, start: 31, end: 33, comments: []>,
    <Token token_type: TokenType.VAR, text: starts, line: 1, col: 41, start: 35, end: 40, comments: []>,
    <Token token_type: TokenType.WITH, text: with, line: 1, col: 46, start: 42, end: 45, comments: []>,
    <Token token_type: TokenType.STRING, text: a, line: 1, col: 50, start: 47, end: 49, comments: []>,
    <Token token_type: TokenType.LIMIT, text: limit, line: 1, col: 56, start: 51, end: 55, comments: []>,
    <Token token_type: TokenType.NUMBER, text: 10, line: 1, col: 59, start: 57, end: 58, comments: []>,
    <Token token_type: TokenType.FROM, text: from, line: 1, col: 64, start: 60, end: 63, comments: []>,
    <Token token_type: TokenType.STRING, text: b, line: 1, col: 68, start: 65, end: 67, comments: []>
]
```

Reply:
You can solve this by using `_parse_table_parts` instead of `_parse_table`; the latter has additional logic for alias parsing which isn't needed for this statement. So, something like this (in [these](https://github.com/tobymao/sqlglot/pull/2845/files#diff-c3dbaf62f01fc7a9baf342a000c2351ebea03abdede048f92d234c44e11c7bfeR627-R632) lines):

```python
                    if self._curr:
                        scope = self._parse_table_parts()
                elif self._curr:
                    scope_kind = "SCHEMA" if this == "OBJECTS" else "TABLE"
                    scope = self._parse_table_parts()
```

And so you'd have this in the return statement:

```python
+                    "starts_with": self._match_text_seq("STARTS", "WITH") and self._parse_string(),
```